pom.xml

```diff
@@ -425,16 +424,10 @@
 
             <!--JAKARTA-->
             <dependency>
-                <groupId>jakarta.mail</groupId>
-                <artifactId>jakarta.mail-api</artifactId>
+                <groupId>com.sun.mail</groupId>
+                <artifactId>jakarta.mail</artifactId>
                 <version>${jakarta.mail.version}</version>
             </dependency>
-            <!--Implementation of the Jakarta Mail API-->
-            <dependency>
-                <groupId>org.eclipse.angus</groupId>
-                <artifactId>angus-mail</artifactId>
-                <version>${angus.mail.version}</version>
-            </dependency>
```

Comment:
Angus started to throw Class Not Found exception so I switched (back) to the reference impl of the mail API. We used it previously but then had to [switch to angus](https://github.com/keycloak/keycloak/commit/bc43e4f4355844a900771e6d215f7c274e77bd4b#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R429-R438) but I forgot the reasoning behind that. Possibly @mabartos remembers.

Reply:
Eclipse Angus is the new way to move forward with the Jakarta Mail - Quarkus team also wants to migrate to Angus (https://github.com/quarkusio/quarkus/issues/26989). Even though I prefer to be aligned as much as possible with Quarkus, in this case, it'd be a move backward as we want to support Jakarta EE 10.

I'd rather resolve the `ClassNotFoundEx` and possibly even upgrade Angus to 2.0.3 and Jakarta Mail API to 2.1.3. @vmuzikar WDYT?

I can take a look at it if you want. 

Also mentioned in https://github.com/keycloak/keycloak/issues/26535#issue-2102544698.

Reply:
Thanks for providing the context. I reverted the changes and angus seem to work now. Not sure what happened but let's wait for the full testsuite to finish.

If the tests pass, I'd prefer upgrading Angus as you suggested as a separate PR as it's not really related to the Quarkus upgrade anymore.